### PR TITLE
Source ip passthrough and whitelisting

### PR DESCRIPTION
**Pull request by MudasarUKCloud**

### CODE REVIEW

File: roles/openshiftpostdeployment/tasks/main.yml [hunk 1]
```diff
@@ -72,6 +72,17 @@
   command: /usr/local/bin/oc set env dc/router-secondary ROUTE_LABELS="router-secondary=true"
   when: multinetwork
 
+- name: Allow Source IP to be recived on primary router from the HAProxy
```
Comment (benbacon) [inline roles/openshiftpostdeployment/tasks/main.yml]: Minor but: *received (including on lines 78 & 82)

**Review by k8soneill — APPROVED** [review]
LGTM

File: roles/openshiftpostdeployment/tasks/main.yml (hunk 1 above)
Comment (k8soneill) [reply roles/openshiftpostdeployment/tasks/main.yml]: Resolved

**Review by stevemul — APPROVED** [review]
LGTM

**Pull request merged**